### PR TITLE
Framework updates

### PR DESCRIPTION
**Pull request by NitishT**
- Add time tracker
- Remove init tests
- Add ENV var to ignore tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,13 +30,14 @@ $ docker build -t minio/mint .
...
-- `ENABLE_HTTPS`   - Optional value when set to 1 sends HTTPS requests on SSL enabled deployment
-- `DATA_DIR`       - Optional value when set uses data from this directory for SDK tests
+Set environment variables to pass test target server details to the docker container. Supported environment variables:
+
+- `SERVER_ENDPOINT`- <IP/URL>:<PORT> of the Minio server on which the tests has to be run. Defaults to [Minio Play Server](play.minio.io:9000/minio/).
+- `ACCESS_KEY`     - Access Key of the server. Defaults to Minio Play Access Key.
+- `SECRET_KEY`     - Secret Key of the server. Defaults to Minio Play Secret Key.
+- `ENABLE_HTTPS`   - Set to 1 to send HTTPS requests on SSL enabled deployment. Defaults to 0.
+- `DATA_DIR`       - Data directory for SDK tests. Defaults to data directory created by `build/data/install.sh` script.
+- `IGNORE_TESTS`   - `;` separated list of SDKs to ignore running. Empty by default.
```
Comment (harshavardhana) [inline README.md]: Comma is easier type than `;`

Comment (NitishT) [inline README.md]: okay

Comment (harshavardhana) [inline README.md]: BTW there are unexpected commits in this PR can you clean it up and squash them?

File: run.sh [hunk 2]
```diff
@@ -21,107 +21,132 @@ log_dir="log"
...
 
-	# run the test
-	./run.sh "$sdk_log_dir/$log_file_name" "$sdk_log_dir/$error_file_name"  && \
+    # run the test
+    ./run.sh "$sdk_log_dir/$log_file_name" "$sdk_log_dir/$error_file_name"  && \
 
-	# move back to top level directory
-	cd ../../..
+    # move back to top level directory
+    cd ../../..
```
Comment (poornas) [inline run.sh]: instead of cd ../../.., it would be better to use cd $root_dir

File: README.md [hunk 3]
```diff
@@ -30,13 +30,14 @@ $ docker build -t minio/mint .
...
-- `ENABLE_HTTPS`   - Optional value when set to 1 sends HTTPS requests on SSL enabled deployment
-- `DATA_DIR`       - Optional value when set uses data from this directory for SDK tests
+Set environment variables to pass test target server details to the docker container. Supported environment variables:
+
+- `SERVER_ENDPOINT`- <IP/URL>:<PORT> of the Minio server on which the tests has to be run. Defaults to [Minio Play Server](play.minio.io:9000/minio/).
+- `ACCESS_KEY`     - Access Key of the server. Defaults to Minio Play Access Key.
+- `SECRET_KEY`     - Secret Key of the server. Defaults to Minio Play Secret Key.
+- `ENABLE_HTTPS`   - Set to 1 to send HTTPS requests on SSL enabled deployment. Defaults to 0.
+- `DATA_DIR`       - Data directory for SDK tests. Defaults to data directory created by `build/data/install.sh` script.
+- `IGNORE_TESTS`   - `,` separated list of SDKs to ignore running. Empty by default.
```
Comment (poornas) [inline README.md]: s/`,` separated list of SDKs to ignore running./',' separated list of SDKs to skip running. 
IGNORE_TESTS could optionally be better named as SKIP_TESTS
An example with this option would be useful as it is not clear here that the sdk naming convention of data directory needs to be followed in the IGNORE_TESTS

**Comment by NitishT** [comment]
Updated @poornas @harshavardhana

**Pull request merged**